ducktape/cluster/json.py

```diff
@@ -100,15 +101,13 @@ def __init__(self, cluster_json=None, *args, **kwargs):
         self._id_supplier = 0
 
     @staticmethod
-    def make_remote_account(ssh_config, externally_routable_ip=None):
+    def make_remote_account(ssh_config, *args, **kwargs):
```

Comment:
This is a big change. Please make sure, all callers of this function are updated to access "externally_routable_ip"  config like a kw?

Reply:
they dont need to, as args also get forwarded.